modules/claims_api/config/initializers/okcomputer.rb

```diff
@@ -76,7 +76,9 @@ def name
 
 class VbmsCheck < BaseCheck
   def check
-    response = Faraday::Connection.new.get(Settings.vbms.url) { |request| request.options.timeout = 20 }
+    connection = Faraday::Connection.new
+    connection.options.timeout = 10
```

Comment:
moved to this structure for setting the timeout because the previous method (line 79 that is removed here) was not actually forcing a timeout like it should.